crates/test/src/lib.rs

```diff
@@ -76,6 +76,12 @@ macro_rules! wasm_bindgen_test_configure {
         pub static __WBG_TEST_RUN_IN_SERVICE_WORKER: [u8; 1] = [0x04];
         $crate::wasm_bindgen_test_configure!($($others)*);
     );
+    (run_in_node $($others:tt)*) => (
+        #[link_section = "__wasm_bindgen_test_unstable"]
+        #[cfg(target_arch = "wasm32")]
+        pub static __WBG_TEST_RUN_IN_NODE: [u8; 1] = [0x05];
+        $crate::wasm_bindgen_test_configure!($($others)*);
+    );
```

Comment:
This should be renamed to contain `experimental` as well.

Not required, but it would be nice if this is documented here:
https://github.com/rustwasm/wasm-bindgen/blob/4596d4a633900a3f474c715aa51937ed0282e205/guide/src/wasm-bindgen-test/browsers.md#L17-L26

Reply:
The `__WBG_TEST_RUN_IN_NODE` here is not actually used anywhere and is only there for the `#[link_section]`.

Reply:
Do you mean the `run_in_node` test specifier?

Reply:
Yes!